ElectronicObserver/Window/Tools/EquipmentUpgradePlanner/UpgradeTree/UpgradeTreeViewModel.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.ObjectModel;
+using CommunityToolkit.Mvvm.DependencyInjection;
+using ElectronicObserver.Common;
+
+namespace ElectronicObserver.Window.Tools.EquipmentUpgradePlanner.UpgradeTree;
+
+public class UpgradeTreeViewModel : WindowViewModelBase
+{
+	public ObservableCollection<UpgradeTreeUpgradePlanViewModel> Items { get; } = new();
```

Comment:
This should always be a single item, so you don't need the observable collection.

Reply:
Actually, that won't work with TreeView.